quantmsutils/mzml/mzml_statistics.py

```
@@ -1,391 +1,615 @@
     RETENTION_TIME,
-    EXPERIMENTAL_MASS_TO_CHARGE,
-    ACQUISITION_DATETIME,
-    MZ_ARRAY,
-    INTENSITY_ARRAY,
-    MONOISOTOPIC_MZ,
-    MAX_INTENSITY,
+    SCAN,
+    SUMMED_PEAK_INTENSITY,
+    PRECURSOR_RT,
+    PRECURSOR_TOTAL_INTENSITY,
+    PRECURSOR_INTENSITY_WINDOW,
```

Ruff (0.8.2)

`quantmsutils.utils.constants.PRECURSOR_INTENSITY_WINDOW` imported but unused

Remove unused import: `quantmsutils.utils.constants.PRECURSOR_INTENSITY_WINDOW`

(F401)